config.yml

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
I understand the removal of the flag in source/common/runtime/runtime_features.cc but how does this config.yml file come into play?